config/default.cfg

```diff
@@ -867,15 +867,15 @@ cfg$gms$c_BaselineAgriEmiRed <- 0
 # (1): iterative adjustment I for tax based on 2030 emissions or for 2150 budget based on 2100 budget result, see core/postsolve.gms for direct algorithms of adjustment
 # (2): iterative adjustment II based on magicc calculated forcing (for both budget and tax runs), see modules/climate/magicc/postsolve.gms for direct algorithms of adjustment
 # (3): [requires 45_carbonprice = "INDC" or "NDC" or "NDCEU" and emiscen = 9] iterative adjustment III for tax based on 2025 or 2030 regionally differentiated emissions, see module/45_carbonprice/<INDC/NDC/NDCEU90>/postsolve.gms for algorithm of adjustment
-# (4): iterative adjustment IV for both budget and tax runs based on CO2 FF&I emissions 2010-2100, see core/postsolve.gms for direct algorithms of adjustment
-# (5): iterative adjustment V for both budget and tax runs based on CO2 emissions 2011-2100, see core/postsolve.gms for direct algorithms of adjustment
+# (4): iterative adjustment IV for both budget and tax runs based on CO2 FF&I emissions 2020-2100, see core/postsolve.gms for direct algorithms of adjustment
+# (5): iterative adjustment V for both budget and tax runs based on CO2 emissions 2020-2100, see core/postsolve.gms for direct algorithms of adjustment
 # (6): iterative adjustment VI for both budget and tax runs based on peak CO2 emissions budget, without changing temporal profile (i.e. with overshoot), see core/postsolve.gms for direct algorithms of adjustment
 # (7): iterative adjustment VII for tax runs based on peak CO2 emissions, with change of temporal tax profile after time of peak budget, aiming for net-zero thereafter, see core/postsolve.gms for direct algorithms of adjustment
 # (9): [REMIND 2.1 default for validation peakBudget runs, in combination with carbonprice = diffCurvPhaseIn2Lin) ] aims at achieving c_budgetCO2 as peakBudget. Scales up and down the original CO2 price path before the peaking year; after the peaking year annual increase by cm_taxCO2inc_after_peakBudgYr. Automatically shifts cm_peakBudgYr to find the correct peaking year for a given .
-#c_budgetCO2 "cumulative CO2|Total emissions budget 2011-2100, Gt CO2"
+#c_budgetCO2 "cumulative CO2|Total emissions budget 2020-2100, Gt CO2"
 # budgets from 1.5°C report, chapter 2, Table 2.2: 67% 1.5°C GSAT budget 2018-2100 = 420 Gt CO2, 67% 2.0°C GSAT budget 2018-2100 = 1170
 # add emissions 2011-2018 = 290 Gt CO2, subtract emissions from feedbacks (e.g. thawing permafrost) 100 Gt CO2
-# budget 2011-2100 for 2°C = 1170 + 290 - 100 = 1350 Gt CO2; for 1.5°C 420 + 290 - 100 = 600 Gt CO2
+# budget 2011-2100 for 2°C = 1170 + 290 - 100 = 1350 Gt CO2; for 1.5°C 420 + 290 - 100 = 600 Gt CO2 !! should we also update this?
```

Comment:
And what about c_budgetCO2 that is set in config files?

Reply:
yes, these have to be updated as well. We will use the numbers from the AR6, which is 400 Gt CO2 for 1.5°C and 1150 Gt CO2 for 2°C targets (both from 2020 on, including 2020)